xarray/core/formatting.py

```diff
@@ -372,7 +372,9 @@ def _mapping_repr(mapping, title, summarizer, col_width=None, max_rows=None):
         max_rows = OPTIONS["display_max_rows"]
     summary = [f"{title}:"]
     if mapping:
-        if len(mapping) > max_rows:
+        len_mapping = len(mapping)
+        if len_mapping > max_rows:
+            summary = [f"{summary[0]} ({max_rows}/{len_mapping})"]
```

Comment:
Does this no longer get the `title` though?

Reply:
Should it be `+=`?

Reply:
It gets the title via summary[0] in the f-string. I did this because I want the number to be displayed on the same row as title, `Attributes: (12/25)`. If we do += the numbers would be shown on a new line.

Reply:
Ah sorry I missed that